src/TestInternalGrains/ActivateDeactivateTestGrain.cs

```diff
@@ -23,13 +46,13 @@ internal class SimpleActivateDeactivateTestGrain : Grain, ISimpleActivateDeactiv
 
         public override async Task OnActivateAsync()
         {
-            this.logger = GetLogger();
+            logger = GetLogger();
```

Comment:
FWIW, the DotNet OSS Coding Style Guide says "avoid  [using] `this.` unless absolutely necessary." (item 4)  No war left to fight on that one! ;)
https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md

But, using `base.` on the other hand is still an area for individual artistic expression .....  :)

However, there are many dark recesses in the Orleans code that pre-date that style guide and haven't been altered in (literally) years. :(

It's a tough call whether bringing legacy code into compliance with the current coding style is best done in one big bang, or incrementally as people touch old code. 
In the past we have tended to do the latter (incremental fix), but if it makes a particular code review too hard then reviewers have also pushed back at times and originator will split.

Sadly the .NET Codeformatter Tool wasn't particularly useful last time we tried it out [early 2015 IIRC].